docs/sources/dashboards/variables/add-template-variables/index.md

```diff
@@ -399,6 +399,10 @@ The syntax above also works with `${__to}`.
 
 You can use this variable in URLs, as well. For example, you can send a user to a dashboard that shows a time range from six hours ago until now: https://play.grafana.org/d/000000012/grafana-play-home?viewPanel=2&orgId=1?from=now-6h&to=now
 
+### `$__url_time_range`
+
+It is also possible to utilise the complete time range by employing the variable `$__url_time_range`. This will result in the expression `from=XXXX&to=XXXX` in which Unix timestamps are used. 
```

Comment:
```suggestion
It is also possible to utilise the complete time range by employing the variable `$__url_time_range`. This will result in the expression `from=XXXX&to=XXXX&timezone=XXX`. 
```

Hi @hemerfaSICKAG , I suggested some changes, since the used unit is not always Unix, and the variable includes time zone too.

Thanks for the contribution!